packages/@lwc/engine-core/src/framework/def.ts

```diff
@@ -237,16 +230,7 @@ export function getComponentInternalDef(Ctor: unknown, name?: string): Component
             );
         }
 
-        let meta = getComponentRegisteredMeta(Ctor);
-        if (isUndefined(meta)) {
-            // TODO [#1295]: remove this workaround after refactoring tests
-            meta = {
-                template: undefined,
-                name: Ctor.name,
-            };
-        }
-
-        def = createComponentDef(Ctor, meta, name || Ctor.name);
+        def = createComponentDef(Ctor, name || Ctor.name);
```

Comment:
why do we need the second argument? if we are going to do that in createComponentDef?